pkg/piecestore/psserver/bucket_refresher.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package psserver
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/dht"
+	"storj.io/storj/pkg/pb"
+)
+
+// Error is a standard error class for this package.
+var (
+	Error                = errs.Class("kademlia bucket refresher error")
+	defaultCheckInterval = flag.Duration("piecestore.kbucket-refresher.check-interval", time.Hour, "number of seconds to sleep between updating the kademlia bucket")
+)
+
+// refreshService contains the information needed to run the bucket refresher service
+type refreshService struct {
+	logger *zap.Logger
```

Comment:
`log`

Reply:
The log lines read a little bit nicer with `log`, e.g. `service.log.Debug(...` vs. `service.logger.Debug(...`

Yeah both are used, but prefer `log`.

Reply:
done